converter_subscribers:
  yolo_detect:
    topic: "/tf_object_detection_zedx_front/obj_detection_msg"
  screen_to_world:  
    topic: "/tf_object_detection_zedx_front/object_detection_world"

converter_publishers:
  output:
    topic: "norfair/input"
    queue_size: 1
